PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/ui/SubscriptionsDialog.java

```diff
@@ -159,9 +156,19 @@ private void createUIComponents() {
         AnActionButton refreshAction = new AnActionButton("Refresh", AllIcons.Actions.Refresh) {
             @Override
             public void actionPerformed(AnActionEvent anActionEvent) {
+                this.setEnabled(false);
```

Comment:
Why there are code changes for authentication? May need update the feature branch?

Reply:
this is not changed by me. i have already aligned my  base branch with develop.